aspnetcore/signalr/get-started-signalr-core/sample/wwwroot/js/chat.js

```diff
@@ -0,0 +1,26 @@
+﻿var connection = new signalR.HubConnection('chathub');
```

Comment:
You're using arrow functions above (`() => {`) which is awesome. All the browsers which support arrow functions also support `let` (for mutable variables) and `const` for immutable variables. Let's update our scripts to use best practices. So if a variable is modified, use `let` instead of `var` and if it isn't modified use `const`.

Reply:
All the variables in this script are immutable (you don't replace the value) so replace all the `var`s with `const`